R/plot_functions.R

```diff
@@ -266,7 +266,7 @@ plot_krona<-function(physeq,output,variable, trim=F){
                "-o", output,
                sep = " "))
   # Run the browser to visualise the output.
-  browseURL(output)
+  # browseURL(output)
```

Comment:
```suggestion
  browseURL(output)
```
The function name is `plot_krona` so for consistency, I prefer to keep the plot by default.
But in a later commit, we could deactivate it with a `plot=FALSE` argument and have a default like this:
```r
if(plot) browseURL(output)
```

Reply:
No problem, this was actually just for me. i forgot this line in the commit.